.github/workflows/test-harness-ariesvcx-ariesvcx.yml

```diff
@@ -41,7 +37,7 @@ jobs:
         with:
           BUILD_AGENTS: "-a aries-vcx"
           TEST_AGENTS: "-d aries-vcx"
-          TEST_SCOPE: "-t @T001-RFC0036,@T002-RFC0036,@T003-RFC0036,@T001-RFC0037,@T001.2-RFC0037,@T001.3-RFC0037,@T001.4-RFC0037,@T001.5-RFC0037,@T003-RFC0037,@T003.1-RFC0037,@T001-RFC0160,@T002-RFC0160,@T003-RFC0160"
+          TEST_SCOPE: "-t @RFC0036,@RFC0037,@RFC0160,@revocation -t ~@wip -t ~@DIDExchangeConnection -t ~@T002-HIPE0011"
```

Comment:
Should the test scope change with VCX to other frameworks -- e.g. VCX to ACA-Py?